src/javascripts/ng-admin/Crud/delete/DeleteController.js

```diff
@@ -41,17 +46,24 @@ define(function () {
     };
 
     DeleteController.prototype.back = function () {
-        this.$location.path(this.entityLabel + '/edit/' + this.entityId);
+        this.$state.go(this.$state.get('edit'), {
```

Comment:
please use `var $state = this.$tate` to avoid repetition. Same in many other controllers.